frontend/src/stores/transfer-history/store.ts

```diff
@@ -17,34 +17,37 @@ export function getStoreName(appVersion: string): string {
   }
 }
 
-export const useTransferHistory = defineStore(getStoreName(getAppMajorVersion(APP_VERSION)), {
-  state: (): TransferHistoryState => ({
```

Comment:
might look like i've changed a lot here but its just due to reformatting. Only change is `APP_VERSION` -> `APP_RELEASE.VERSION`